docs/how-to-guides/demos/isis-demos.md

```diff
@@ -0,0 +1,36 @@
+# ISIS Demos
+
+<script src="https://asc-public-docs.s3.us-west-2.amazonaws.com/common/scripts/isis-demos/jquery-3.7.1.min.js"></script>
+<link href="../../../css/isis-demos.css" media="all" rel="stylesheet"/>
+
+
+<script type="text/javascript">
+if (typeof window.isisDemosLoaded == 'undefined') {
+    var isisDemosLoaded = true;
+    $.getScript("https://asc-public-docs.s3.us-west-2.amazonaws.com/common/scripts/isis-demos/easeljs-0.8.1.min.js").done( function(s,t) { $.getScript("../../../js/isisDemos.js");});
+}
+</script>
+
+## Image Data Size Calculator
+<div class="app-container" id="isis-image-data-size"></div>
+
+## Cube Demo
+<div class="app-container" id="isis-cube"></div>
+
+## Pixel Demo
+<div class="app-container" id="isis-pixels"></div>
+
+## Multiplier Demo
+<div class="app-container" id="isis-multiplier"></div>
+
+## Subpixels Demo
+<div class="app-container" id="isis-subpixels"></div>
+
+## Special Pixels Demo
+<div class="app-container" id="isis-special-pixels"></div>
+
+## Destripe Demo
+<div class="app-container" id="isis-destripe"></div>
+
+## Seam Removal Demo
+<div class="app-container" id="isis-seams"></div>
```

Comment:
Where were these originally taken from? Do they have analogous locations in the other docs?

Reply:
To make this page, I took them from a similar page which just listed all the demos.  I remember a long time ago when we were initially moving docs over, some demos were taken out of the pages they were on... I don't remember which pages those were.